python/triton/language/core.py

```diff
@@ -461,6 +425,16 @@ def is_int(self):
     def is_bool(self):
         return self.is_int1()
 
+    def kind(self):
+        # Return int value following the type ordering bool < integer < fp
+        if self.is_bool():
+            return 0
```

Comment:
Would it be possible to make it not hardcoded?

Reply:
what are you proposing exactly?

Reply:
Something like Python Enum